Core/scripts/dirac-install.py

```diff
@@ -1678,7 +1679,9 @@ def discoverModules(modules):
            ('x:', 'external=', 'external version'),
            ('  ', 'createLink', 'create version symbolic link from the versions directory. This is equivalent to the \
            following command: ln -s /opt/dirac/versions/vArBpC vArBpC'),
-           ('  ', 'scriptSymlink', 'Symlink the scripts instead of creating wrapper')
+           ('  ', 'scriptSymlink', 'Symlink the scripts instead of creating wrapper'),
+           ('  ', 'userEnvVariables',
```

Comment:
It seems that to work correctly you should replace with:

('  ', 'userEnvVariables=',

Reply:
Correct. Done.